org.libredirect.frontends-manager.yml

```diff
@@ -0,0 +1,52 @@
+id: org.libredirect.frontends-manager
+runtime: org.gnome.Platform
+runtime-version: '42'
+command: libredirect-frontends-manager
+rename-icon: libredirect-frontends-manager
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-nightly
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
you say you use it for docker, but what if docker doesn't exist.

Why can't you instead bundle the docker command that connect to the docker socket?

Reply:
Well that would be better. I'll try to implement that.

Reply:
Apparently with Docker Desktop there's no docket socket. Even [Whaler](https://flathub.org/apps/com.github.sdv43.whaler) isn't working. I use docker to download images and run/stop them.

Reply:
I also have a way to detect if docker doesn't exist and notify the user.

Reply:
but it this package require something external like docker, then the point is moot.

also why not podman or more opensource friendle solution

Reply:
> Apparently with Docker Desktop there's no docket socket

it needs to be exposed inside the flatpak. Which is also terrible, but still better.